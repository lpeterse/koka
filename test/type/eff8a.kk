fun forever(f) {
  f();
  print("hi");
  forever(f);
}