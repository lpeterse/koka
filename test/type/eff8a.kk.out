forever     : forall<a,b,e> (() -> <div,console|e> a) -> <div,console|e> b
